docs/app/devbox.json

```diff
@@ -1,6 +1,12 @@
 {
-  "packages": [],
+  "packages": [
+    "nodejs-16_x",
```

Comment:
Can just do `nodejs`

Reply:
FYI, reason I chose this:

<img width="705" alt="image" src="https://user-images.githubusercontent.com/544948/202816259-468b009e-d65d-478a-9c4c-e2ffb757b27f.png">